src/share/poudriere/jail.sh

```diff
@@ -1045,6 +1077,7 @@ SETNAME=""
 XDEV=0
 BUILD=0
 GIT_DEPTH=--depth=1
+BUILD_PKGBASE=0
```

Comment:
How is this getting set? In Poudriere.conf? It should have an example there or should be an added cmdline option.

Reply:
with -B on the jail creation (already upstreamed).